{
 "bad": [
  "errno",
  "errno.ENOENT",
  "neptune.new",
  "neptune.new.internal.utils.verify_type",
  "neptune.new.types.File.as_image",
  "neptune.new.types.File.as_pickle",
  "neptune.new.types.FileSeries",
  "neptune_xgboost.__init__.get_versions",
  "os.path.basename",
  "os.path.dirname",
  "os.path.realpath",
  "sys",
  "sys.exc_info",
  "sys.platform",
  "xgboost.callback.TrainingCallback"
 ],
 "deps": {
  "errno": [],
  "json": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "matplotlib": [
   "matplotlib-base/conda-forge/linux-64/matplotlib-base-3.3.3-py36h159efd9_0"
  ],
  "neptune": [],
  "neptune_xgboost": [],
  "os": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "re": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "subprocess": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "sys": [],
  "warnings": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "xgboost": [
   "py-xgboost/conda-forge/linux-64/py-xgboost-1.1.1-py37hc8dfbb8_0",
   "xgboost/conda-forge/linux-64/xgboost-0.6a2-py27_2"
  ]
 }
}